api/v1/types.go

```diff
@@ -7,48 +7,121 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+type StateType string
+
+const (
+	StateTypeHealthy      StateType = "Healthy"
+	StateTypeUnhealthy    StateType = "Unhealthy"
+	StateTypeInitializing StateType = "Initializing"
+	StateTypeDegraded     StateType = "Degraded"
```

Comment:
Please add comment for `StateTypeInitializing`

```suggestion
	StateTypeDegraded     StateType = "Degraded"
	StateTypeInitializing StateType = "Initializing"
```